### ISSUE TITLE
Add possibility to specify base path via env var for dashboard

### ISSUE THREAD

**Issue by vladtkachuk** [sorry-cypress/sorry-cypress#487]
- [x] I have searched issues for similar feature requests

## What is this feature about (1 sentence)  

Base path should be runtime configurable via the *env variable*.

## Why is it needed? What is the value? For whom do we build it?  

If one wants to expose the dashboard not on the domain root but let's say at `my-domain.com/sorry-cypress/dashboard` the dashboard will try to load the resources from root, e.g., `my-domain.com/fonts.css` while it should be `my-domain.com/sorry-cypress/dashboard/fonts.css`.

## High-level feature overview

Basically above

## Other

If there is already a way to do this, I would be grateful for the hint, so far I had to make (not very nice) changes to the code.

**Comment by sforner405** [comment]
Would still be important for us
